site/web/assets/js/classification.js

```diff
@@ -0,0 +1,140 @@
+var insertClassification = function (classificationDetails) {
+  var count = 1;
+  for (var i = 0; i < classificationDetails.length; i++) {
+    if (classificationDetails[i][2] == null) {
+      $('#category-' + count + '').append('<option value="' + classificationDetails[i][3] + '">' + classificationDetails[i][3] + '</option>');
```

Comment:
printing HTML values and attribute values needs escaping. It's much safer to do this using the DOM functions instead of constructing a string. (Please fix that throughout the JS code.)

Reply:
I am replacing the string with $("<option>").val(classificationDetails[i]["name"]).html(classificationDetails[i]["name"]). Is this okay or do you want any other method to do this